yktrace/src/swt.rs

```diff
@@ -126,6 +129,10 @@ unsafe fn yk_swt_start_tracing_impl() {
 }
 
 /// Record a location into the trace buffer if tracing is enabled on the current thread.
+///
+/// This function is separate from `__yk_swt_rec_loc_impl` to prevent the fast path of tracing
```

Comment:
I think we could simplify this to "This function calls a fast or a slow implementation depending on the platform".

Reply:
That is not quite the reason. Does the new comment make it clearer?